### ISSUE TITLE
Bot for zulip communication for anitaborg

### ISSUE THREAD

**Issue by Tlazypanda** [Tlazypanda/testing_bots#10]
Discussion issue for bot designed to facilitate onboarding for anitaborg zulip workspace.

**Comment by Tlazypanda** [comment]
@Frenzyritz13 can we take this up further here? :sweat_smile: Please share anything that can be useful for us and let's define the scope of what all our bot can do. Thanks :v:

**Comment by Frenzyritz13** [comment]
@Tlazypanda Sure. 
So we need a bot to help with community engagement and outreach, especially to onboard newcommers. We need our bot to do the following actions for the time being, and will work on the scope based on the user interaction.
- Send a short welcome message about how the streams and subscription to the different streams works, everytime a user signs up. This can also be a private message since bots are allowed to send private messages.
- Upload and send a the onboarding bingo, and preferably prompt them to complete the tasks that are possible in zulip.
- Prompt them to contact the people based on how they want to contribute.

Problems as of now:
- It requires a seperate server running at all times.
- The trigger activity is unclear.

**Comment by Tlazypanda** [comment]
Thanks @Frenzyritz13 for breaking it down :raised_hands: This is the api we will be making use of https://chat.zulip.org/api/real-time-events for the tasks mentioned. Thanks :v:

**Comment by Frenzyritz13** [comment]
@Tlazypanda got it. But my concern regarding the server remains the same. Point me to the right resource to get a better, if possible please!

**Comment by Tlazypanda** [comment]
Yeah that is inevitable as of now I am also not sure if we have enough resources for that since zulipbot is also pending because of the same issue 

Tagging @mayburgos for further clarification on whether we can get a server for this ...

We can setup a personal server on heroku but the free one sleeps in 30 mins if not used. For better servers we will have to pay...so we can proceed with the free one for now till we have resources available :sweat_smile:

**Comment by Frenzyritz13** [comment]
@Tlazypanda Yap, we can do that. We can set it up with the resources at hand and deploy it later at any suitable place

**Comment by mayburgos** [comment]
@Tlazypanda I will let you know as soon as I know. I've been having a lot of blockers with lack of access in AWS. :/
